### DISCUSSION TITLE
R code for translating GenTB geneID output to entrez geneIDs that can be used by David

### DISCUSSION THREAD

**Discussion by sandergoossens** [TORCH-Consortium/MAGMA#32] [Q&A]
Hi, as discussed today with @abhi18av we would expand the pipeline with GenTB and David to ultimately annotate our variants too (pathway, gene ontology, ...).

One step between genTB and david is to translate geneIDs, i wrote a code for that in R.

Where do i put this?

**Comment by abhi18av** [comment]
For the time being @sandergoossens , you can put the code in the comment here as markdown. https://guides.github.com/features/mastering-markdown/ 


We'll merge it in the main code once it has been tested and ready for inclusion.

**Reply by abhi18av** [reply]
Just realized, you've already added the code here https://github.com/abhi18av/xbs-nf/commit/ca59fa058c734afca7685081b97fde00cedc42dd - thanks!

Will test this once done with current iteration on the code.
